webapp/src/Controller/Jury/ContestController.php

```diff
@@ -330,10 +335,23 @@ public function indexAction(Request $request): Response
                 } else {
                     $timeValue = Utils::printtime($time, $timeFormat);
                     $timeTitle = Utils::printtime($time, 'Y-m-d H:i:s (T)');
+                    $dayInSeconds = 24*60*60;
+                    if($timeField === 'activate' && $contest->getStarttimeEnabled()) {
+                        if(Utils::difftime($contestdata['starttime']['value'], $time)/$dayInSeconds>1.0) {
```

Comment:
Why not just `Utils::difftime($contestdata['starttime']['value'], $time)>$dayInSeconds`?

Reply:
Yes, that is better.